server/cluster_info.go

```diff
@@ -450,6 +450,9 @@ func (c *clusterInfo) handleRegionHeartbeat(region *core.RegionInfo) error {
 		if region.ApproximateSize != origin.ApproximateSize {
 			saveCache = true
 		}
+		if region.ApproximateRows != origin.ApproximateRows {
+			saveCache = true
```

Comment:
Is the `ApproximateRows` too sensitive to change?

Reply:
I'm not sure, should we @nolouch @disksing ?

Reply:
I think it should be ok as long as they are only saved to cache (not to etcd).